Runtime/Core/CinemachineCore.cs

```diff
@@ -72,7 +72,11 @@ public static CinemachineCore Instance
         /// <summary>Delegate for overriding Unity's default input system.
         /// If you set this, then your delegate will be called instead of
         /// System.Input.GetAxis(axisName) whenever in-game user input is needed.</summary>
+#if ENABLE_LEGACY_INPUT_MANAGER
         public static AxisInputDelegate GetInputAxis = UnityEngine.Input.GetAxis;
+#else
+        public static AxisInputDelegate GetInputAxis = null;
```

Comment:
What happens if it gets called?  No one is checking for null.  I think it should be an empty function instead.